### PR TITLE
Unit tests round 2 - Additional tests for LADDERS models

### PR DESCRIPTION
**Pull request by binrai**

### CODE REVIEW

File: models/DM/dm_views_config.yml
Comment (shuclu) [inline models/DM/dm_views_config.yml:1]: @binrai the right syntax for relationships TO field should be:

         to: source('dm_table_data', 'CASE_PROVIDER')

if you have wrong file name or wrong syntax, the dbt test most of the time won't complain, but it won't get compiled either. 

to know if it's compiled, you can see it's included in the the result list or check if the file exists and gets updated in e.g. target/compiled/data_and_analytics_dbt_bha/models/DM_DB/DM/Views/dm_views_config.yml/relationships_VW_CLINIC_PROVID_bdfa5056da537d1ef8ed52f2e59e1f13.sql

Comment (shuclu) [reply models/DM/dm_views_config.yml:1]: also make sure that all other new dbt unit tests do get complied, either you see it in the result list or in the file

Comment (binrai) [reply models/DM/dm_views_config.yml:1]: To make sure I'm understanding this correctly:


1) I do see the relations it in the list of compiled files under the following path:
`target/run/data_and_analytics_dbt_bha_ladders/models/DM/dm_views_config.yml` 

 + Example output from the recent run. 
`data_and_analytics_dbt_bha_ladders/target/run/data_and_analytics_dbt_bha_ladders/models/DM/dm_views_config.yml/relationships_VW_CLINICS_CREAT_6fcd13e31db27a6ce5a5be56c58ca339.sql`

2) I also see the following output in the terminal after running the dbt test command.
`4 of 12 PASS relationships_VW_CLINICS_CREATE_UPDATE_PARENT_CASE_ID__CASE_ID___source_dm_table_data_CASE_PROVIDER_`

@shuclu  Can you confirm I'm looking at the right places? Happy to update the syntax for the relationship but want to make sure I'm looking at the right places for future as well.

+ also confirmed other tests the same away as step 1 and 2 above.

Comment (binrai) [reply models/DM/dm_views_config.yml:1]: Also, I did go ahead and change the syntax and confirmed its working.

**Review by shuclu — APPROVED** [review]

**Pull request merged**